user/store.go

```diff
@@ -102,6 +103,13 @@ func NewDB(ctx context.Context, db *sql.DB) (*DB, error) {
 			WHERE id = $1
 		`),
 
+		setUserRole: p.P(`
+			UPDATE users
+			SET
+				role = $2
+			WHERE id = $1
+		`),
```

Comment:
nit: just one-line this since it's so short

```suggestion
		setUserRole: p.P(`UPDATE users SET role = $2 WHERE id = $1`),
```